cypress/e2e/hermioneAccount.spec.cy.js

```diff
@@ -0,0 +1,56 @@
+import { faker } from '@faker-js/faker';
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  const accountNumber = '1001';
+  const startBalance = 5096;
+  const randomDeposit = faker.number.int({ min: 1000, max: 9999 });
+  const currentBalance = startBalance + randomDeposit;
+  const withdrawAmount = faker.number.int({ min: 1000, max: 5000 });
+  const balanceAfter = currentBalance - withdrawAmount;
+
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('should provide the ability to work with Hermione\'s bank account', () => {
+    cy.contains('button', 'Customer Login').click();
+    cy.get('#userSelect').select('Hermoine Granger');
```

Comment:
Ensure that the name 'Hermoine Granger' matches exactly with the option available in the dropdown. If there is a typo or mismatch, the test will fail.